src/components/NSLookupTool/NSLookupTool.tsx

```diff
@@ -7,50 +7,54 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
-const title = "NsLookup";
-const description_userguide =
-    "The nslookup command is a tool used to query Domain Name System (DNS) servers and retrieve information about a specific domain or IP address." +
+/**
+ * Title and Description of the tool
+ */
+const title = "NSLookup";
+const description_Userguide =
```

Comment:
Change this constant title to description. 
```suggestion
const description =
```